### ISSUE TITLE
Finetune YOLO model more to increase mAP score

### ISSUE THREAD

**Issue by HCA97** [HCA97/Mosquito-Classifiction#22]
The training loss curves of YOLO don't look good, I think the reason might we are doing fine tuning and the initial lr might be too much or too little.

![results](https://github.com/HCA97/Mosquito-Classifiction/assets/24678502/22917f8b-a1f5-46ed-b700-40c52db27e18)
